### DISCUSSION TITLE
OM to matlab

### DISCUSSION THREAD

**Discussion by Yahiaassam** [OpenModelica/OpenModelica#9096] [Q&A]
Hello, I am currently working on a project to know the consumption of H2 under different constraints such as vessel pressure and temperature, and in order to integrate my model into Matlab (FMU export), I want to have inputs and outputs in Matlab for that I can change the pressure and the temperature to have in exit a consumption of H2.
The question is how do I do in OM to have these I/O interfaces ?

for the moment I only have my model in a single block, without the I/O interfaces.

Thank's for your answers

**Comment by lucas-mior** [comment]
You can declare your inputs with the `input` keyword, and outputs with `output` keyword. Then export your FMU and import into matlab.
